javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,50 @@
+// Variables for candy prices, amount to spend, summary of candies
+
+const candyPriceSweet = 0.5;
+const candyPriceChocolate = 0.7;
+const candyPriceToffee = 1.1;
+const candyPriceChewing = 0.03;
+let boughtCandyPrices = [];
+let sum = 0;
+const amountToSpend = Math.random() * 100;
+// This function calculates price of the candy & push our candy price to array
+function addCandy(candyType, weight) {
+  if (candyType === "sweet") {
+    priceCandy = weight * candyPriceSweet;
+    boughtCandyPrices.push(priceCandy);
```

Comment:
```suggestion
    sum += priceCandy;
```
same for the other instances below

Reply:
Thanks a lot, it saved me from a lot of unnecessary code ^^